ext/curl/tests/bug71144.phpt

```diff
@@ -1,14 +1,13 @@
 --TEST--
 Bug #71144 (Sementation fault when using cURL with ZTS)
+Since Curl 7.62, CURLOPT_DNS_USE_GLOBAL_CACHE has no effect, and is
+silently ignored.
```

Comment:
I think this additional info should go into a [`--DESCRIPTION--` section](https://qa.php.net/phpt_details.php#description_section):
```suggestion
--DESCRIPTION--
Since Curl 7.62, CURLOPT_DNS_USE_GLOBAL_CACHE has no effect, and is
silently ignored.
```

Reply:
Thank you! Fixed now.